packages/esm-patient-vitals-app/src/vitals/vitals-overview.component.tsx

```diff
@@ -70,39 +71,69 @@ const VitalsOverview: React.FC<VitalsOverviewProps> = ({ patientUuid, pageSize,
     };
   }, [patient, t, excludePatientIdentifierCodeTypes?.uuids]);
 
-  const tableHeaders = [
-    { key: 'date', header: t('dateAndTime', 'Date and time'), isSortable: true },
+  const tableHeaders: Array<VitalsTableHeader> = [
     {
-      key: 'temperature',
+      key: 'dateRender',
```

Comment:
Is the idea to repackage this logic as a custom hook, similar to what you did with conditions above?

Reply:
Yes @denniskigen, I'll move this hook inside the framework such that all the data tables can use this similar implementation.
The new need from the developer will be to add the `sortFunc` for every header in the tableHeaders.

If you have a better idea than this, I'm all open to hear it.
Thansk!